app-emulation/cri-tools/Manifest

```diff
@@ -0,0 +1 @@
+DIST cri-tools-1.18.0.tar.gz 5295512 BLAKE2B 9dda4a401b858e98aa60521d3268d8f6ff6a2c1bd2c219a0e5e9e5360fba58d41e97258c636da26ee57bbfc3286798c74107c7de92d3e3398bdf53b7d9078e94 SHA512 849b496e6d19eec29e0e56f0b37c094649311b80514f319b0c68c6519e3ddd48e1973c3d288144db998826676416644ec705b7ea0916c21898b40928ea800af1
```

Comment:
Any reason for picking up 1.18 instead of 1.20? The package seems to be unmaintained on Gentoo side, which maybe is why this package is lagging.

Reply:
> Any reason for picking up 1.18 instead of 1.20? The package seems to be unmaintained on Gentoo side, which maybe is why this package is lagging.

I just happened to copy Gentoo ebuild, which was 1.18. That's all.

Ideally, I would like to update the version to 1.20.
However, when I changed the version to 1.20, then the binary size of `crictl` gets much bigger, from 19M to 30M.
Not sure I like it. Let me think how we can avoid.

Reply:
Fortunately crictl 1.19 is only 21M. So I simply updated to 1.19.
Added commit messages about that.

Another option to resolve the size issue is to get crictl included in torcx tarballs.
However, it is not so a clean way, because cri-tools is not an existing package of the "torcx world". cri-tools is not containerd.
We would need to touch all over the place in torcx, it does not look good in the long term.
I would not like to add another dimension of complexity to torcx.